JS/Lesson26/onder/script.js

```diff
@@ -1,14 +1,27 @@
-/* 
-Elements to use:
-li (.post)
-h2 (.post-title)
-p (.post-body)
-*/
-
-const URL = 'https://jsonplaceholder.typicode.com/posts';
-function fetchPosts() {
+function editPost(id) {
+    window.location.href = `edit.html?id=${id}`;
```

Comment:
The url isn't correct, should be `EditPost/edit.html?id=${id}` according to the current project structure